### DISCUSSION TITLE
Loot Lookup

### DISCUSSION THREAD

**Discussion by Deathlysmiles** [runelite/runelite#16114] [Q&A] [answered]
Is it possible to have the look up be able to look up items and have a list of monsters with drop rates instead? For example I type in bronze bar and it shows dwarf (1/18.29) etc.

**Comment by Enriath** [comment, marked as answer]
The wiki has all this information, and can be quickly accessed through the button next to the minimap.

**Reply by Deathlysmiles** [reply]
Having the wiki open also defeats the purpose of Loot Lookup so I don't see this point being relevant. I want to use it through the client just like loot lookup instead of having the wiki open 24/7.

**Reply by Enriath** [reply]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.
